examples/src/examples/user-interface/world-screen.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import * as pc from 'playcanvas/build/playcanvas.js';
+import { AssetLoader } from '../../app/helpers/loader';
+import Example from '../../app/example';
+
+class WorldScreenExample extends Example {
+    static CATEGORY = 'User Interface';
+    static NAME = 'World Screen';
+
+    load() {
+        return <>
+            <AssetLoader name="checkboard" type="texture" url="static/assets/textures/checkboard.png" />
+            <AssetLoader name='font' type='font' url='static/assets/fonts/courier.json' />
+        </>;
+    }
+
+    example(canvas: HTMLCanvasElement, assets: { checkboard: pc.Asset, font: pc.Asset }): void {
+
+        // Create the application with input and start the update loop
+        const app = new pc.Application(canvas, {
+            mouse: new pc.Mouse(document.body),
+            touch: new pc.TouchDevice(document.body),
+            elementInput: new pc.ElementInput(canvas)
+        });
+        app.start();
+
+        // Create an Entity with a camera component
+        const camera = new pc.Entity();
+        camera.addComponent("camera", {
+            clearColor: new pc.Color(1, 0, 0)
+        });
+        camera.rotateLocal(-30, 0, 0);
+        camera.translateLocal(0, 0, 7);
+        app.root.addChild(camera);
+
+        // Create an Entity for the ground
+        const material = new pc.StandardMaterial();
+        material.diffuse = pc.Color.WHITE;
+        material.diffuseMap = assets.checkboard.resource;
+        material.diffuseMapTiling = new pc.Vec2(50, 50);
+        material.update();
+
+        const ground = new pc.Entity();
+        ground.addComponent("render", {
+            type: "box",
+            material: material
+        });
+        ground.setLocalScale(50, 1, 50);
+        ground.setLocalPosition(0, -0.5, 0);
+        app.root.addChild(ground);
+
+        // Create an entity with a light component
+        const light = new pc.Entity();
+        light.addComponent("light", {
+            type: "directional",
+            color: new pc.Color(1, 1, 1),
+            castShadows: true,
+            intensity: 1,
+            shadowBias: 0.2,
+            shadowDistance: 16,
+            normalOffsetBias: 0.05,
+            shadowResolution: 2048
+        });
+        light.setLocalEulerAngles(45, 30, 0);
+        app.root.addChild(light);
+
+        // Create a capsule entity to represent a player in the 3d world
+        const entity = new pc.Entity();
```

Comment:
you could create few of them into an array, going around .. it would be a handy test case for maybe a batching / text ordering and similar at some point.

Reply:
maybe make the text clickable and change the color of it?

Reply:
Uu I like that! Updated.

Reply:
I just wonder .. if there's some way to have a progress bar for each player as well, this could be even more useful example.